<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
   background-color:  #222831;
}

QToolButton{
  border: none;
}

QToolButton::pressed{
 border: 1px solid #76ABAE; 
 border-radius: 17px;
}

#video_progress::groove:horizontal{
      border: 1px solid #f7f7f7;
      height: 12px;
      background: #FFFBE6;
      margin: 2px 0;
}

#video_progress::handle:horizontal{
       background: #1A3636;
       border: 1px solid #5c5c5c;
       width: 25px;
       border-radius: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="80,20">
    <item>
     <widget class="QVideoWidget" name="video_widget" native="true">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Drag and drop a video file to play, or right click and load a video file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottom_layout">
      <item>
       <layout class="QVBoxLayout" name="bottom_main_layout">
        <item>
         <widget class="QSlider" name="video_progress">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="bottom_main_layout_progress" stretch="0,0">
          <item>
           <widget class="QLabel" name="video_length_label">
            <property name="font">
             <font>
              <family>Academy Engraved LET</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="video_progress_label">
            <property name="font">
             <font>
              <family>Academy Engraved LET</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="bottom_main_layout_controls">
          <item>
           <widget class="QToolButton" name="backward_button">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/Resources.qrc">
              <normaloff>:/Icons/Previous.png</normaloff>:/Icons/Previous.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="play_button">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/Resources.qrc">
              <normaloff>:/Icons/Play.png</normaloff>:/Icons/Play.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="forward_button">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/Resources.qrc">
              <normaloff>:/Icons/Next.png</normaloff>:/Icons/Next.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="mute_button">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/Resources.qrc">
              <normaloff>:/Icons/Mute.png</normaloff>:/Icons/Mute.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="volume_slider">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>QtMultimediaWidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
